### DISCUSSION TITLE
Additional Wallet Support

### DISCUSSION THREAD

**Discussion by stevew00dy** [sunflower-land/sunflower-land#1284] [Ideas]
As we focus on mass adoption we need to consider two things:

1, How to remove as much friction as possible
3, How to simplify the process

When we think about our target audience (non crypto native - non tech savvy casual gamers) we have to consider the path of least resistance for them to be able to start, enjoy and progress within the game.

Right now there are many solutions to Web3 onboarding but none are considered simple and easy. 

 My suggestion is to create a process where users can start to play without the worry about setting up Defi Wallets and connecting to Layer 2 networks. This can all come in the future. 
 
 After extensive research I have found the best overall global solution to be Coinbase. 
 
 They offer a suite of products that will enable users to deposit using a debit or credit card, create a wallet from their coinbase account and also list us on their centralised exchange. 
 
 I recommend we keep Metamask as this is the preferred option but we also use Coinbase as an alternative. 
 
 We have also experienced lots of interest from other wallet providers but my main concern is that too many options will become overwhelming. Therefore I would like to keep the options to just 3 
 
 Metamask
 Coinbase
 Wallet Connect
 
 As an additional option I would like to consider 
 
 Other Wallets (this would expand to give other options like crypto.com, 1inch and other wallets in the future)

**Comment by stevew00dy** [comment]
Here is an example UI of how it could look (designed with our brand in mind)
![image](https://user-images.githubusercontent.com/5225886/181853867-cefa4d61-4f1d-4f6a-8794-e25b331ccc47.png)

**Comment by JcEii** [comment]
Sir @stevew00dy like this kind of UI? 

![wallet](https://user-images.githubusercontent.com/105348036/182022375-70e1d00f-9f01-45f8-af37-8ba218062183.png)
![mema wallet](https://user-images.githubusercontent.com/105348036/182022378-ae81bfc1-faa1-4f7b-8545-81dcc6ffea84.png)
 
 I agree with the additional wallet support to allow new or old players to connect their own online wallets to the game. For example, in my country, we can connect PayPal or maybe Gcash to the game, and the player will decide which wallet will transact (like sync and mint some nft). It will either be metamask or an additional wallet (such as Coinbase wallet, walletconnect, and other wallets).
